dashboard/src/pages/EventVolunteers.vue

```
@@ -17,7 +17,7 @@
     <!-- VOLUNTEERS GRID -->
     <div class="mt-4 grid grid-cols-1 gap-4 md:grid-cols-2 lg:grid-cols-3">
       <Card v-for="member in event.doc.event_members" :key="member.name" :title="member.full_name">
-        <template v-if="member.email != session.user && member.role != 'Lead' && isLead()" #actions>
+        <template v-if="member.email != session.user && member.role != 'Lead' && isCoreTeam()" #actions>
```

ESLint

[error] Replace `·v-if="member.email·!=·session.user·&&·member.role·!=·'Lead'·&&·isCoreTeam()"·#actions` with `⏎··········v-if="member.email·!=·session.user·&&·member.role·!=·'Lead'·&&·isCoreTeam()"⏎··········#actions⏎········`

(prettier/prettier)